.github/workflows/ci.yml

```diff
@@ -13,7 +13,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: cachix/install-nix-action@v18
-      - run: curl --retry 5 --retry-all-errors -sfL https://direnv.net/install.sh | bash
+      - run: |
+          curl -v --retry 5 --retry-all-errors -sfL https://direnv.net/install.sh | sed 's/curl/curl -v --retry 5 --retry-all-errors --header "authorization: Bearer ${{ secrets.GITHUB_TOKEN }}"/g' | bash
```

Comment:
Oof. Might want to toss a [yaml anchor](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/) on this first one, and then use that for the other jobs. Save us some copy-pasta.

Reply:
if i can prove this works, i've had a lot of false hope :)

Reply:
github says nope:
`The workflow is not valid. .github/workflows/ci.yml: Anchors are not currently supported. Remove the anchor 'install-direnv'`

Reply:
I moved to multiline to make it more readable. Haven't had a failure after many runs so going to merge this and 🤞